book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(author, title)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(book)
+    @rentals.push(book)
+  end
+end
```

Comment:
- [ ] The implementation of this method is incorrect. Kindly note that the purpose of this method is to allow a book create a rental using the `person` and `date`. Now in the rental class initialize method we can see that instantiating a rental also adds the rental to the respective `rentals` arrays of book and person, so we don't need to do that here. Kindly modify this method to something like this:

  ```
  def add_rental(person, date)
    Rental.new(date, person, self)
  end
  ```